src/github/nag.rs

```diff
@@ -464,24 +464,31 @@ fn evaluate_nags() -> DashResult<()> {
     Ok(())
 }
 
-fn execute_ffcp_actions(issue: &Issue, disposition: FcpDisposition) {
-    if !is_rfc_repo(&issue) { return; }
+fn can_ffcp_close(issue: &Issue) -> bool {
+    SETUP.ffcp_auto_close(&issue.repository)
+}
+
+fn can_ffcp_postpone(issue: &Issue) -> bool {
+    SETUP.ffcp_auto_postpone(&issue.repository)
+}
 
+fn execute_ffcp_actions(issue: &Issue, disposition: FcpDisposition) {
     match disposition {
         FcpDisposition::Merge => {
             // TODO: This one will require a lot of work to
             // auto-merge RFCs and create the tracking issue.
         },
-        FcpDisposition::Close => {
+        FcpDisposition::Close if can_ffcp_close(issue) => {
```

Comment:
that's a cool pattern that's new to me -- someone should write up all the fun things you can do in rust bindings